lib/syntax.ml

```diff
@@ -178,7 +185,12 @@ let resolve_aggregations =
     | ResInparam _| ResChoices (_, _)
     | ResAggValue _ | ResInTupleList _ -> res_expr
   in
-  handle ~is_agg:false  
+  handle ~is_agg:false 
+  
+let resolve_ambiguous_columns all_schema final_schema = 
```

Comment:
the name resolve_ is confusing - it is not resolving anything, it is just adding alias columns to schema